config/initializers/active_record_bug_patch.rb

```diff
@@ -0,0 +1,24 @@
+# This is a monkey patch for a bug that is in Rails 5.0.0. When a JSON column
+# in a database table is set to null it is incorrectly serialized to a string
+# value of "null" rather than the underlying database format of NULL.
+#
+# This has already been merged into Rails: https://github.com/rails/rails/pull/25670
+# and should be released for Rails 5.0.1
+#
+# FIXME: If this is still here and Rails 5.0.1 or later is being used, you can
```

Comment:
It's good to add something like:
```ruby
raise "Is this still required?" if Gem::Version.new(Rails.version) > Gem::Version.new("5.0.0")
```

That way when we upgrade Rails we're forced to check if this is still valid.